server/package.json

```diff
@@ -17,6 +17,8 @@
     "mongoose": "^5.4.10",
     "passport": "^0.4.0",
     "passport-jwt": "^4.0.0",
+    "socket.io": "^2.2.0",
+    "socket.io-stream": "^0.9.1",
```

Comment:
Is this dependency for the server or for the client?

Reply:
for the server